packages/console-client/src/kiota-client/api/extensibility/extensions/index.ts

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable @typescript-eslint/ban-tslint-comment */
+/* tslint:disable */
+/* eslint-disable */
+// Generated by Microsoft Kiota
+// @ts-ignore
+import { type AdditionalDataHolder, type ApiError, type BaseRequestBuilder, type Parsable, type ParsableFactory, type ParseNode, type RequestConfiguration, type RequestInformation, type RequestsMetadata, type SerializationWriter } from '@microsoft/kiota-abstractions';
```

Comment:
import type {} from "kiota" doesn't work?

Reply:
This is generated by kiota, so not in our control. It is possible to add the license header? 🤔

Reply:
this file is autogenerated